containers/orchestration/app/handlers.py

```diff
@@ -12,24 +10,95 @@
 }
 
 
+class ServiceHandlerResponse:
+    """
+    Wrapper class that encapsulates all the information the `call_apis`
+    function needs to handle processing one service's output as another
+    service's input. This class captures the return status of a service
+    (status code), what data it sent back (its message content), and
+    whether this data is fit for processing by another service (whether
+    call_apis should continue). Some of the DIBBs services either do not
+    return altered data to the caller, or return a 200 status code but
+    still flag an error with their input, so capturing these nuances
+    allows `call_apis` to be completely agnostic about the response
+    content a service sends back.
+    """
+
+    def __init__(
+        self, status_code: int, msg_content: str | dict, should_continue: bool
+    ):
+        self._status_code = status_code
+        self._msg_content = msg_content
+        self._should_continue = should_continue
+
+    @property
+    def status_code(self) -> int:
+        """
+        The status code the service returned to the caller.
+        """
+        return self._status_code
+
+    @status_code.setter
+    def status_code(self, code) -> None:
```

Comment:
I think we should have a type hint on `code` and add the param to the doc string.

```suggestion
    def status_code(self, code: int) -> None:
```